src/beeswarm.jl

```diff
@@ -0,0 +1,50 @@
+# ---------------------------------------------------------------------------
+# Beeswarm plot
+@recipe function f(::Type{Val{:beeswarm}}, x, y, z; trim::Bool=false, side::Symbol=:both)
+    if !(side in [:both :left :right])
+        warn("side (you gave :$side) must be one of :both, :left, or :right")
+        side = :both
+        info("side set to :$side")
+    end
+    x, y = Float64[], Float64[]
+    glabels = sort(collect(unique(x)))
```

Comment:
Part of the code here is identical across beeswarm, violin and boxplot - I haven't checked carefully, but would it not be possible to extract a general function?